src/web/src/utils.js

```diff
@@ -294,3 +300,28 @@ export const exportScheduleToIcs = (selectedCourses) => {
     })}_Schedule`
   );
 };
+
+/**
+ * Export The Course Schedule To An Image Format, Probably PNG.
+ * Can Come Back Here + Add More File Format Options, Such As JPEG, ...
+ */
+export const exportScheduleToImage = (selectedCourses, currentSemester) => {
+  // Handle Special Case Where No Selected Courses On Schedule.
+  if(selectedCourses.length == 0){
+    alert("No Courses Found For Export To Image Data.")
+    return;
+  }
+  // Obtain Schedule Element Defined In Schedule.Vue File + Run Export To PNG. 
+  var currentScheduleNode = document.getElementById('allScheduleData')
+  domtoimage.toPng(currentScheduleNode, {bgcolor: currentScheduleNode.getAttribute('isDarkModeOn')})
```

Comment:
instead of trying to get the background color of the schedule with an attribute, maybe we can just pass it in as a parameter to `exportScheduleToImage`?

Reply:
Hmm I can try to do that, but that should not be causing the Color to be Transparent though, right?

Reply:
Well if the attribute name doesn't end up being `isDarkModeOn` for some reason, maybe getAttribute will return undefined

Reply:
Hmm but I have been doing Inspect Element and that is the Attribute Name.
But when I output it, it has NULL as opposed to the Color.